api/rest/v1/bulk_find_entities.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: default
+title: Find DCIDs
+parent: REST (v1)
+grand_parent: API
+nav_order: 1
+published: true
+permalink: /api/rest/v1/bulk/find/entities
+---
+
+# /v1/bulk/find/entities
+
+Find the DCID of an entity.
+
+Given the name of the entity, this endpoint searches for an entry in the Data Common knowledge graph and returns the DCID of the best match. For example, you could query for "San Francisco, CA" or "San Francisco" to find that its DCID is `geoId/0667000`. You can also provide the type of entity (country, city, state, etc.) to disambiguate (Georgia the country vs. Georgia the US state).
+
```

Comment:
We probably want to add some prominent disclaimer about the accuracy and provide tips to improve mapping.  Like:

IMPORTANT: This API relies on name-based geocoding and is prone to inaccuracies.  One common pattern is ambiguous place names that existing in different countries, states, etc.. For example there is at least one popular city called "Cambridge" in both UK and USA. Thus, for more precise results, please provide as much context in the description as possible.  For example, to resolve Cambridge in USA, pass "Cambridge, MA, USA" if you can.

NOTE: This API now only supports place entities

Reply:
Added disclaimers!

New screenshot:
![Screen Shot 2022-11-10 at 3 34 59 PM](https://user-images.githubusercontent.com/4034366/201228279-64b610dd-f7aa-4761-be59-9aaccb7665aa.png)

Reply:
Maybe say something that we now only support place types, but will extend in the future as the DC graph grows/expands/etc.

Also, move the NOTE above the IMPORTANT?